mephisto/tools/scripts.py

```diff
@@ -234,8 +237,21 @@ def build_custom_bundle(custom_src_dir, run_config: DictConfig):
 
     IGNORE_FOLDERS = {os.path.join(prebuild_path, f) for f in IGNORE_FOLDERS}
     build_path = os.path.join(prebuild_path, "build", "bundle.js")
+
+    if "force_rebuild" not in kwargs:
+        warnings.warn(
+            "Consider adding the force_rebuild property to your hydra config and passing it into build_custom_bundle()\nSteps on how to do this: \nhttps://github.com/facebookresearch/Mephisto/issues/811",
+            stacklevel=2,
+        )
+
+    if "post_install_script" not in kwargs:
+        warnings.warn(
+            "Consider adding the post_install_script property to your hydra config and passing it into build_custom_bundle()\nSteps on how to do this: \nhttps://github.com/facebookresearch/Mephisto/issues/811",
+            stacklevel=2,
+        )
```

Comment:
New warning messages:
They link to a closed issue that describes how to fix this